templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -94,7 +94,7 @@ if [[ "$TEST" = 'bindings' || "$TEST" = 'publish' ]]; then
   gem install --both ./{{ plugin_snake }}_client-0.gem
   cd ..
 {%- endif %}
-  ruby $REPO_ROOT/.ci/assets/bindings/test_bindings.rb
+  SSL_CERT_FILE=/usr/local/share/ca-certificates/pulp_ca.crt ruby $REPO_ROOT/.ci/assets/bindings/test_bindings.rb
```

Comment:
Ruby doesn't use certifi lib, so I had to use an OpenSSL env var to point to the CA